de.dlr.sc.virsat.fdir.core/src/de/dlr/sc/virsat/fdir/core/matrix/Matrix.java

```diff
@@ -162,14 +162,25 @@ public void setStatePredRates(double[][] statePredRates) {
 	}
 
 	@Override
-	public double multiply(double[] vector, double[] result) {
+	public void multiply(double[] vector, double[] result) {

```

Comment:
I think we should make the class abstract so we dont have to provide all these dummy implementaitons

Reply:
Whole class got removed